src/test/java/org/reactome/release/otherIdentifiers/PostStepChecksTest.java

```diff
@@ -4,25 +4,19 @@
 import org.gk.model.ReactomeJavaConstants;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
-
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.*;
 
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
 import static org.junit.jupiter.api.Assertions.*;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({PostStepChecks.class})
-@PowerMockIgnore({"org.apache.logging.log4j.*", "javax.management.*", "javax.script.*",
```

Comment:
Do the tests still work with the Mockito runner instead of the PowerMock runner?"

Reply:
Yup, I ran them on my local machine.